tsconfig.json

```diff
@@ -2,37 +2,27 @@
   "compilerOptions": {
     "allowJs": true,
     "esModuleInterop": true,
+    "emitDeclarationOnly": true,
     "allowSyntheticDefaultImports": true,
     "jsx": "react-jsx",
     "noEmit": false,
     "noImplicitAny": false,
     "forceConsistentCasingInFileNames": true,
     "strict": true,
-    "typeRoots": ["./node_modules/@types"],
     "sourceMap": true,
     "inlineSources": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "module": "ESNext",
+    "module": "Preserve",
     "target": "ES2022",
     "declaration": true,
-    "moduleResolution": "node",
-    "types": ["jest", "node", "react"],
-    "baseUrl": "./",
-    "paths": {
-      "@/*": ["src/*"]
-    },
```

Comment:
**note:** This `paths` config is replaced with the import map in our package manifest.